package.json

```diff
@@ -10,10 +10,13 @@
     "es6-promise": "^4.2.8",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-modal": "^3.11.2",
     "react-redux": "^7.2.0",
+    "react-responsive-modal": "^5.0.2",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.1",
-    "redux": "^4.0.5"
+    "redux": "^4.0.5",
+    "simple-react-modal": "^0.5.1"
```

Comment:
3 packages for modal is used use anyone which fits our requirement and **uninstall ** others